### ISSUE TITLE
re-include paf~ as an external in 'extra' or upload it to deken.

### ISSUE THREAD

**Issue by porres** [pure-data/pure-data#1685]
Hi, I'm revisiting this and keeping this issue so it's on my radar and I don't forget about it. 

I have once tried to make a PR to re-include paf~, see <https://github.com/pure-data/pure-data/pull/364>. What happened in this closed PR is that the old code for paf~ needed some cleaning and rewriting... for reference, see --> https://github.com/pure-data/pure-data/pull/290#issuecomment-419738348 and --> https://github.com/pure-data/pure-data/pull/364#issuecomment-420848191

In short, @millerpuckette says he thinks paf~ doesn't belong in Pd anymore and maybe we should upload it to deken, right? Also, he seems to have a working code privately some where. So I'm pinging @millerpuckette to see what we can do about this. If this is supposed to be just an upload external now, I can help maintaining it, building it and supporting it.

cheers
